packages/core/index.ts

```diff
@@ -1,158 +1 @@
- * @param {String} [version] version identifier x.y.z for the contract. Defaults to "latest".
- */
-const getTruffleContractTest = (contractName: string, web3: Web3 | undefined, version = "latest"): TruffleContractI => {
-  return version === "latest"
-    ? ((artifacts as GenericArtifacts).require(contractName) as TruffleContractI)
-    : getTruffleContractProd(contractName, web3, version);
-};
-
-/**
- * @notice Gets the contract address. This method will automatically be exported instead of getAdress in the case that
- * this is being used in a truffle test context.
- * @param {String} contractName Name of the UMA contract whose address is to be retrieved.
- * @param {Integer} networkId  Network ID of the network where that contract is deployed.
- * @param {String} [version] version identifier x.y.z for the contract. Defaults to "latest".
- */
-const getAddressTest = (contractName: string, networkId: number, version = "latest"): string => {
-  const truffleContract = getTruffleContractTest(contractName, undefined, version);
-
-  const { _provisioner } = artifacts as GenericArtifacts;
-  if (truffleContract.networks[networkId]) {
-    return truffleContract.networks[networkId].address;
-  } else if (_provisioner?._deploymentAddresses[contractName] && _provisioner?._networkConfig.chainId === networkId) {
-    // In the production hardhat case, there is no networks object, so we fall back to hardhat's global list of deployed addresses as long as hardhat's network id matches the one passed in.
-    // Note: this is a bit hacky because it depends on internal hardhat details.
-    return _provisioner._deploymentAddresses[contractName];
-  } else if (hardhatTestingAddresses[contractName]) {
-    // If running tests in hardhat, check if there is a testing address set.
-    return hardhatTestingAddresses[contractName];
-  } else {
-    throw new Error(`No address found for contract ${contractName} on network ${networkId}`);
-  }
-};
-
-/**
- * @notice Gets the contract abi. This method will automatically be exported instead of getAbi() in the case that
- * this is being used in a truffle test context.
- * @param {String} contractName Name of the UMA contract whose abi is to be retrieved.
- * @param {String} [version] version identifier x.y.z for the contract. Defaults to "latest".
- */
-const getAbiTest = (contractName: string, version = "latest"): any => {
-  const truffleContract = getTruffleContractTest(contractName, undefined, version);
-  return truffleContract.abi;
-};
-
-const globalArtifactsExists = Boolean(((global as unknown) as { artifacts: any }).artifacts);
-const getAbi = globalArtifactsExists ? getAbiTest : getAbiProd;
-const getAddress = globalArtifactsExists ? getAddressTest : getAddressProd;
-const getTruffleContract = globalArtifactsExists ? getTruffleContractTest : getTruffleContractProd;
-
-export { getAbi, getAddress, getTruffleContract, findContractVersion };
+export { findContractVersion } from "./src/FindContractVersion";
```

Comment:
This is the only function we're still using in core. Not too concerned about relying on it. Maybe down the road, we can move it out.

Reply:
ye, could easily move it common